webui/downloads.js

```diff
@@ -769,7 +770,22 @@ var DownloadsUI = (new function($)
 			return ' <span class="label label-priority label-info">priority: ' + priority + '</span>';
 		}
 	}
-	
+
+	this.buildEncryptedLabel = function(parameters) {
+		var isEncrypted = false;
+
+		for (var i=0; i < parameters.length; i++) {
+			if (parameters[i]['Name'] === '*Unpack:Password' &&
+				parameters[i]['Value'])
+			{
+				isEncrypted = parameters[i]['Value'];
+				break;
+			}
+		}
+		return isEncrypted !== false ?
```

Comment:
Here isEncrypted can be either boolean (false) or a string. Is it ok to compare string with `false`?